client/elife-theme/src/components/Button.js

```diff
@@ -0,0 +1,33 @@
+import { css } from 'styled-components'
+
+const extraSmall = css`
+  font-size: 11px;
+  line-height: calc(${props => props.theme.gridUnit} * 2);
+  padding: ${props => props.theme.gridUnit};
```

Comment:
Why not use `th`?

Reply:
I think I started from other component and completely forgot about `th`.